Neo4j.Driver/Neo4j.Driver.Tests/ConnectionPoolTests.cs

```diff
@@ -665,8 +665,8 @@ public class NumberOfInUseConnections
             public void ShouldReturnZeroAfterCreation()
             {
                 var uri = new Uri("localhost:7687");
-                var connectionSettings = new ConnectionSettings(uri, AuthTokens.None, Config.DefaultConfig);
-                var poolSettings = new ConnectionPoolSettings(1, 1, TimeSpan.MaxValue, TimeSpan.MaxValue);
+                var connectionSettings = new ConnectionSettings(AuthTokens.None, Config.DefaultConfig);
+                var poolSettings = new ConnectionPoolSettings(1, 1, Config.Infinite, Config.Infinite, Config.Infinite);
```

Comment:
Found System.Threading.TimeOut static class which declares InfiniteTimeSpan static field which is also using -1. No change needed then, but we may point out this in the docs.